pgjdbc/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
This should not be part of this PR.

Reply:
actually, I'm not seeing this here ?

Reply:
The file pgjdbc/.gitignore is being added in the commit with message "moving the url-modifiying test out of BaseDataSourceTest (#1039)".

Rebase your PR atop the latest master and diff against that to see the changes.